CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixes
 
+- Fix span recording for `NSFileManager.createFileAtPath` starting with iOS 18, macOS 15 and tvOS 18 (#4634)
```

Comment:
`m`: I think we should mention that you have to enable a feature for this, which is still experimental and will be enabled by default in the future.

Reply:
Added context how to enable it